scripts/sysman/power.yml

```diff
@@ -227,6 +233,7 @@ name: GetProperties
 details:
     - "The application may call this function from simultaneous threads."
     - "The implementation of this function should be lock-free."
+    - "1.7": "Note: This function is deprecated and replaced by $sPowerGetPowerProperties."
```

Comment:
`- "1.7": "[DEPRECATED] Use $sPowerGetPowerProperties."`

Reply:
removing this line since we need to first define the new function